.gitignore

```diff
@@ -1,6 +1,8 @@
 # dependencies
-node_modules
-vendor
+/node_modules
```

Comment:
Wouldn't we want the more-generic non-`/`-prefixed version? We could theoretically end up committing nested modules, with this change.